app/assets/utils/_custom.scss

```diff
@@ -4,4 +4,8 @@
 .amount {
   text-align: right !important;
   white-space: nowrap;
-}
\ No newline at end of file
+}
+
+.inlineButton {
```

Comment:
for consistency I would have like this `.inline-button` instead of camelCase

Reply:
Done